### PR TITLE
Status Bar Link 

### PR DESCRIPTION
**Pull request by Randy424**
Signed-off-by: randybrunopiverger <rbrunopi@redhat.com>
Screen Shot 2021-06-07 at 1 46 47 PM

### CODE REVIEW

File: src/AcmProgressTracker/AcmProgressTracker.tsx [hunk 1]
```diff
@@ -114,6 +125,19 @@ export function AcmProgressTracker(props: AcmProgressTrackerProps) {
                                     {step.statusSubtitle}
                                 </Text>
                             </TextContent>
+                            {step.link && (
+                                <AcmButton
+                                    className={classes.button}
+                                    variant="link"
+                                    icon={<ExternalLinkAltIcon size="sm" />}
+                                    iconPosition="right"
+                                    onClick={() => step.link?.linkTo()}
```
Comment (Randy424) [inline src/AcmProgressTracker/AcmProgressTracker.tsx:134]: @jamestalton  Any idea how to target this line in my testing? I am using `userEvent.click()` and it results in the callback being fired off, but jest is still finding that this line is not covered...

**Review by jamestalton — APPROVED** [review]
lgtm

File: src/AcmProgressTracker/AcmProgressTracker.tsx [hunk 2]
```diff
@@ -114,6 +125,22 @@ export function AcmProgressTracker(props: AcmProgressTrackerProps) {
                                     {step.statusSubtitle}
                                 </Text>
                             </TextContent>
+                            {step.link && (
+                                <AcmButton
+                                    className={classes.button}
+                                    variant="link"
+                                    icon={<ExternalLinkAltIcon size="sm" />}
```
Comment (showeimer) [inline src/AcmProgressTracker/AcmProgressTracker.tsx:132]: I thought the button would automatically handle the icon size?

File: src/AcmProgressTracker/AcmProgressTracker.tsx [hunk 3]
```diff
@@ -114,6 +125,22 @@ export function AcmProgressTracker(props: AcmProgressTrackerProps) {
...
+                                    iconPosition="right"
+                                    onClick={() => {
+                                        /* istanbul ignore next */
+                                        window.open(step.link?.linkUrl)
+                                    }}
+                                >
+                                    <Text className={classes.link} component="a">
```
Comment (showeimer) [inline src/AcmProgressTracker/AcmProgressTracker.tsx:139]: Why is the Text component needed here, and making it an anchor tag?

Comment (showeimer) [reply src/AcmProgressTracker/AcmProgressTracker.tsx:139]: Maybe you were trying to get the actually hyperlink behavior, if so, you just need to set the `AcmButton` prop to have a `isInline` prop and it should look like a regular hypertext link

File: src/AcmProgressTracker/AcmProgressTracker.tsx [hunk 4]
```diff
@@ -71,7 +79,10 @@ const useStyles = makeStyles({
     },
     button: {
         padding: 0,
-        fontSize: 'inherit',
+    },
+    link: {
+        paddingLeft: '25px',
+        fontSize: 'small',
```
Comment (showeimer) [inline src/AcmProgressTracker/AcmProgressTracker.tsx:85]: small?

**Comment by Randy424** [comment]
PR is regarding: https://github.com/open-cluster-management/backlog/issues/13131

**Pull request merged**